ydb/core/tx/datashard/local_kmeans.cpp

```diff
@@ -77,6 +82,7 @@ class TLocalKMeansScanBase: public TActor<TLocalKMeansScanBase>, public NTable::
 
     std::vector<TProbability> MaxRows;
     std::vector<TString> Clusters;
+    std::vector<ui64> Counts;
```

Comment:
```suggestion
    std::vector<ui64> AggregatedClustersPrevCount;
```

Reply:
It's not aggregated counts, it's cluster counts

Reply:
Changed